bot/exts/moderation/infraction/_utils.py

```diff
@@ -180,9 +188,15 @@ async def notify_infraction(
         expires_at = "Never"
         duration = "Permanent"
     else:
+        if "last_applied" in infraction:
+            origin = infraction["last_applied"]
+        else:  # Fallback for previous API versions without `last_applied`
+            log.trace(f"No last_applied for infraction {infraction}, using inserted_at time.")
+            origin = infraction["inserted_at"]
```

Comment:
Same as in the first comment

Reply:
Fixed in [97b940a](https://github.com/python-discord/bot/pull/2234/commits/97b940a24ac90fd7ec8405b84e0b4f419c95d31a)